picard/ui/searchdialog.py

```diff
@@ -146,6 +147,8 @@ def trigger_search_action(self):
 
 class CoverWidget(QtWidgets.QWidget):
 
+    shown = pyqtSignal()
```

Comment:
Correct me if I am wrong but doesn't this make the scrolled signal based fetching of CAA redundant?

Reply:
> Correct me if I am wrong but doesn't this make the scrolled signal based fetching of CAA redundant?

That's a good point, and you may be right, i need to verify this.

Reply:
According to my experiments, it is required, i'll see if we can improve that later, but for now, i'd let this as is.